atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/cleaner/KeyValueServicePuncherStore.java

```diff
@@ -88,12 +114,15 @@ private void tryInitialize() {
         keyValueService.createTable(AtlasDbConstants.PUNCH_TABLE, new TableMetadata(
                 NameMetadataDescription.create(ImmutableList.of(
                         new NameComponentDescription.Builder()
-                                .componentName("time")
+                                .componentName("time_rounded_to_day")
```

Comment:
This looks like an artifact of when the resolution was previously set to 1 day.

Reply:
fixed